tests/acceptance/test_organization_releases_v2.py

```diff
@@ -25,9 +29,37 @@ def test_no_access(self):
         self.browser.wait_until_not(".loading")
         self.browser.snapshot("organization releases v2 - no access")
 
-    def test(self):
+    def test_list(self):
         with self.feature(FEATURE_NAME):
+            self.create_release(project=self.project, version="1.0")
             self.browser.get(self.path)
             self.browser.wait_until_not(".loading")
-            # TODO(releasesv2): data is for now randomly hardcoded in the UI - this snapshot will always be different, turned off until finished api
-            # self.browser.snapshot("organization releases v2 list")
+            self.browser.snapshot("organization releases v2 - with releases")
+            # TODO(releasesV2): add health data
+
+    def test_detail(self):
+        with self.feature(FEATURE_NAME):
+            release = self.create_release(project=self.project, version="1.0")
+            self.browser.get(self.path + release.version)
+            self.browser.wait_until_not(".loading")
+            self.browser.snapshot("organization releases v2 - detail")
+            # TODO(releasesV2): add health data
+
+    def test_detail_pick_project(self):
+        with self.feature(FEATURE_NAME):
+            release = self.create_release(
+                project=self.project, additional_projects=[self.project2], version="1.0"
+            )
+            self.browser.get(self.path + release.version)
+            self.browser.wait_until_not(".loading")
+            self.browser.snapshot("organization releases v2 - detail - pick project")
+
+    def test_detail_global_header(self):
+        with self.feature(FEATURE_NAME):
+            release = self.create_release(
+                project=self.project, additional_projects=[self.project2], version="1.0"
+            )
+            self.browser.get(u"{}?project={}".format(self.path + release.version, self.project.id))
+            self.browser.wait_until_not(".loading")
+            self.browser.click('[data-test-id="global-header-project-selector"]')
+            self.browser.snapshot("organization releases v2 - detail - global project header")
```

Comment:
Do we need snapshots here? If we have snapshots elsewhere I would just make some assertions that content is there.

Reply:
This is snapshotting that new feature of globalSelectionHeader project picker where we see only specified projects and a custom footer message saying `Only projects with this release are visible.` I promised this to @markstory a long time ago.

Reply:
👍 I would add ^ description somewhere :)